packages/js/src/utils/angular.ts

```diff
@@ -52,6 +54,32 @@ export type LogtoAngularConfig = {
    * @default true
    */
   includeReservedScopes?: boolean;
+  /**
+   * Login hint indicates the current user (usually an email address or a phone number).
+   * @link SignInUriParameters.loginHint
+   */
+  loginHint?: SignInUriParameters['loginHint'];
```

Comment:
I’m just curious, why is there a special parameter file for Angular?

Reply:
It's used for converting Logto parameters into the ones supported by `augular-auth-oidc-client` library, as we don't have a native Angular SDK.